ui/admin/app/styles/app.scss

```diff
@@ -1078,3 +1078,15 @@
     margin-bottom: 0;
   }
 }
+
+.change-state {
+  margin-right: 1rem;
+}
+
+.dropdown-radio-btn {
+  min-width: 20rem;
```

Comment:
What is the reason to give a min width? Do we need this dropdown to have a min width by design? or its just to keep same as before?

Reply:
thanks for the call out, we didn't need that, I removed it!